src/main/cpp/t04_both2.cpp

```diff
@@ -23,9 +23,38 @@
 
 #include "t04_both2.h"
 #include <iostream>
-
+#include <set>
 using namespace std;
 
 int t04_both2() {
+	set <int> a;
+	set <int> b;
+	set <int> d;
+
+	int N, M;
+	int number = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'number' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4721709)